go-controller/pkg/ovn/master.go

```diff
@@ -593,15 +600,22 @@ func (oc *DefaultNetworkController) syncNodesPeriodic() {
 		delete(chassisHostNameMap, nodeName)
 	}
 
-	staleChassis := []*sbdb.Chassis{}
-	for _, v := range chassisHostNameMap {
-		staleChassis = append(staleChassis, v)
+	staleChassis := make([]*sbdb.Chassis, 0, len(chassisHostNameMap))
+	staleChassisIds := make([]string, 0, len(chassisHostNameMap))
+	for _, chassis := range chassisHostNameMap {
+		staleChassis = append(staleChassis, chassis)
+		staleChassisIds = append(staleChassisIds, chassis.Name)
 	}
 
 	if err = libovsdbops.DeleteChassis(oc.sbClient, staleChassis...); err != nil {
 		klog.Errorf("Failed Deleting chassis %v error: %v", chassisHostNameMap, err)
 		return
 	}
+
+	if err = libovsdbops.DeleteChassisTemplateRecord(oc.nbClient, staleChassisIds...); err != nil {
+		klog.Errorf("Failed Deleting chassis template vars %v error: %v", chassisHostNameMap, err)
```

Comment:
unrelated to this PR, but I wonder why this func doesn't return errors.. :/

Reply:
I don't know for sure but I guess it's because we call it from a ticker.